test/integration/Dockerfile

```diff
@@ -23,6 +24,10 @@ COPY gomplate /bin/gomplate
 COPY mirror /bin/mirror
 COPY meta /bin/meta
 COPY aws /bin/aws
+COPY k8s /bin/k8s
+COPY k8ssvc/generate.sh /tmp/
+COPY k8ssvc/openssl.cnf /tmp/
+RUN /tmp/generate.sh
```

Comment:
Can the script be run from the test itself, at runtime, instead of build-time? If it's run at build-time the test will only work when the test is run inside a container.